src/api/lib/wizard.rb

```diff
@@ -1,6 +1,15 @@
 require 'rexml/document'
 
 class Wizard
+  def self.guess_version(name, tarball)
```

Comment:
With this change this method is now public instead of private. Do we want that? :confused: